mlos_bench/mlos_bench/tests/launcher_run_test.py

```diff
@@ -93,8 +93,7 @@ def test_launch_main_app_opt(root_path: str, local_exec_service: LocalExecServic
     """
     _launch_main_app(
         root_path, local_exec_service,
-        # TODO: Reset --max_iterations to 3 after fixing the optimizer
-        "--config mlos_bench/mlos_bench/tests/config/cli/mock-opt.jsonc --trial_config_repeat_count 3 --max_iterations 9",
+        "--config mlos_bench/mlos_bench/tests/config/cli/mock-opt.jsonc --trial_config_repeat_count 3 --max_iterations 3",
```

Comment:
Can we also use this PR to rename "max_iterations" to "num_of_suggested_configs" or "max_num_configs" or something like that?
#715

Reply:
sure, let's do that

Reply:
We already did the breaking change in the last one with how `--max-iterations` no longer is multiplied.  I'd say we finish it and cut a new release now.  Or we could finish this one and do it in the next PR.  Thoughts?

Reply:
I am always in favor of multiple small PRs 😸

Reply:
As always, it depends.  I prefer them too, unless it's too hard to see what's happening or there's breakage in the middle.  Anyways, let's move it to the next one.  Thanks!

Reply:
ok, I've updated it to `--max_suggestions` - hope you're ok with this!